yarn.lock

```diff
@@ -1683,6 +1683,14 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-conformance-griffel@9.0.0-beta.19":
+  version "9.0.0-beta.19"
+  resolved "https://registry.yarnpkg.com/@fluentui/react-conformance-griffel/-/react-conformance-griffel-9.0.0-beta.19.tgz#3998ff3764c685929091dcaac5b85ad1353feda8"
+  integrity sha512-wJSdW5Z08Jm8jQRqXbb1fj+Dn3ifqR2gTabx3ghKnpWvwSMj8s3EVPoeoaO8phCCyIP7c5XtQDYhc+KBW3K8Cw==
+  dependencies:
+    "@griffel/react" "^1.5.2"
+    tslib "^2.1.0"
+
```

Comment:
There shouldn't be any changes to yarn.lock -- this should go away once you change to use `*` for the version in package.json. Adding this comment to make sure that happens :)